crates/frontend/src/pages/dimensions.rs

```diff
@@ -50,6 +50,7 @@ pub fn dimensions() -> impl IntoView {
                 _ => Some(json!(function_name.replace('"', ""))),
             };
             let mandatory = row["mandatory"].as_bool().unwrap_or(false);
+            let dimesnion_name: StoredValue<_> = StoredValue::new(row_dimension.clone());
```

Comment:
Why do we need this? You can use `row_dimension` directly right?

Reply:
No, you cant use row_dimension directly since the value gets moved inside the delete handler

Reply:
Then why use `StoredValue ` can directly clone and use right?

Reply:
And add delete modal for the delete confirmation, the component is already there.

Reply:
Done